src/linkedList.js

```diff
@@ -0,0 +1,112 @@
+class Node {
+  constructor(data) {
+    this.data = data,
+    this.next = null
+  }
+}
+
+export default class linkedList {
+  constructor(){
+    this._length = 0,
+    this.head = null,
+    this.tail = null
+  }
+
+  insert(data) {
+    let node = new Node(data)
+
+    if(this.head === null){
+      ++this._length
+      return this.head = this.tail = node
+    }
+    let currentNode = this.head
+
+    while(currentNode.next){
+      currentNode = currentNode.next
+    }
+      ++this._length
+      currentNode.next = node
+    return this.tail = node
+  }
+  
+  size() { return this._length }
+
+  getHeadNode(){ return this.head }
+
+  getTailNode(){ return this.tail }
+
+  contains(data) {
+    let currentNode = this.head
+
+    while(currentNode.next) {
+      if(currentNode.data === data) return true
+      currentNode = currentNode.next
+    }
+    if(this.getTailNode().data === data) return true
```

Comment:
Here's another case of `if( true ) return true` - you can just return the result of the boolean expression directly:
```js
return this.getTailNode().getData() === data
```